src/panel_material_ui/widgets/menus.py

```diff
@@ -323,6 +323,37 @@ class MenuButton(MenuBase, _ButtonBase):
     _item_keys = ['label', 'icon', 'color', 'items']
 
 
+class SplitButton(MenuBase, _ButtonBase):
```

Comment:
For some unknown reason the Panel MUI `SplitButton` does not look like the MUI `SplitButton`. The Panel version does not have the clear split between button and dropdown.

https://github.com/user-attachments/assets/5035d115-ba33-4c16-b713-b978adc23088

Reply:
Thank you for giving this a try! I think the behavior you are seeing comes from me not updating the reference notebook to use `SplitButton` instead of `MenuButton`. So I think you ended up checking a `MenuButton` in case you copy paste the added reference notebook code 😅 Last commit should change that